### ISSUE TITLE
Evaluate different Dropout parameter

### ISSUE THREAD

**Issue by michaelwerger** [michaelwerger/keras_based_line_identification#7]
Test tf.keras.layers.Dropout(d) with d = [0.1, 0.2, ... 0.9]
